### DISCUSSION TITLE
Why multiple Adapters?

### DISCUSSION THREAD

**Discussion by mattbue** [nzymedefense/nzyme#678] [Questions]
Hi,
yesterday i found nzyme. It looks really nice and I already plan to use it.
Until now there is one question on which I couldn't get an answer on...
What are reasons to use multiple wifi adapter on one system?

Thanks
mattbue

**Comment by justinfreitag** [comment]
Bandit location (via trackers). You can put both a tracker and the nzyme service/leader on a single device, but then you lose the benefits of location.
